enum.c

```diff
@@ -238,21 +254,28 @@ count_all_i(RB_BLOCK_CALL_FUNC_ARGLIST(i, memop))
 }
 
 /*
- *  call-seq:
- *     enum.count                 -> int
- *     enum.count(item)           -> int
- *     enum.count { |obj| block } -> int
+ * call-seq:
+ *   count -> integer
+ *   count(object) -> integer
+ *   count {|element| ... } -> integer
+ *
+ * Returns the count of elements, based on an argument or block criterion, if given.
```

Comment:
```suggestion
 * Returns the count of elements, based on an argument or block criterion.
```

"If given" seems to indicate that the block can be used optionally with the argument version, yet the both are alternative forms (either argument or block).